frontend/packages/dev-console/src/components/topology/operators/operators-data-transformer.ts

```diff
@@ -59,15 +59,18 @@ export const getOperatorTopologyDataModel = (
         if (_.isEmpty(operator)) {
           operator = operatorBackedServiceKindMap[nodeResourceKind];
         }
+
+        const csvIcon =
+          _.get(operatorBackedServiceKindMap[nodeResourceKind], 'spec.icon.0') ||
+          _.get(operator, 'spec.icon.0');
```

Comment:
```suggestion
        const csvIcon = _.get(operator, 'spec.icon.0');
```
You can simply do this because if you see above there is a check for operator and if it is empty it is assigned to `operatorBackedServiceKindMap[nodeResourceKind]` so we can get the icon from `operator` itself in both the cases.

Reply:
Both could exist. We don't attempt to look at `operatorBackedServiceKindMap[nodeResourceKind]` if we found the operator via the ownerUid. This will consider both.

Reply:
So you mean to say if we get the operator via the ownerUid then also for the icon we will give precedence to operatorBackedServiceKindMap[nodeResourceKind] and only if we dont get the icon from there then we will try to get it from operator that we got via the ownerUid?

Reply:
I could reverse the order to change the precedence. I left it this way thinking the original author may have purposefully chosen to use the icon from the kind map.

Reply:
Got it. I think this order should be fine since we should show the icon for the particular operator backed service and if there is none then use the operator icon.

Reply:
I would prefer that we move away from using lodash `get` everywhere and use optional chaining instead.